s3transfer/manager.py

```diff
@@ -586,6 +586,10 @@ def wait(self):
             # If Keyboard interrupt is raised while waiting for
             # the result, then exit out of the wait and raise the
             # exception
+            if transfer_coordinator:
```

Comment:
Are you sure this is safe to assume `transfer_coordinator` is defined?  What is `KeyboardInterrupt` is raised before entering the for loop, but after the try?  Wouldn't you get an `UnboundLocalError`?